app/assets/src/components/views/SampleViewV2/ReportTable.jsx

```diff
@@ -586,8 +586,8 @@ class ReportTable extends React.Component {
           rowData.genus
             ? getOr(nullValue, path, rowData)
             : sortDirection === "asc"
-              ? limits[0]
-              : limits[1],
+            ? limits[0]
```

Comment:
linter fixed this

Reply:
We could try to a full code lint update, to just do all at once. Do you want to try that on a follow up?

Reply:
Yeah I think that's a great idea!